File: Iniciante/URI 2930.c
#include <stdio.h>
 
int main() { 
    int e, d;

    scanf("%d %d", &e, &d);
    if (e > d) {
        printf("Eu odeio a professora!\n");
    } else if ((d - e) >= 3) {
        printf("Muito bem! Apresenta antes do Natal!\n");
    } else if ((e + 2) < 24) {
        printf("Parece o trabalho do meu filho!\nTCC Apresentado!\n");
    } else {
        printf("Parece o trabalho do meu filho!\nFail! Entao eh nataaaaal!\n");
    }
    
    return 0;
}

File: Iniciante/URI 2930.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var line = lines.shift().split(" ");
var E = parseFloat(line[0]);
var D = parseFloat(line[1]);

if (E > D) {
    console.log('Eu odeio a professora!');
} else if ((D - E) >= 3) {
    console.log('Muito bem! Apresenta antes do Natal!');
} else if ((E + 2) < 24) {
    console.log('Parece o trabalho do meu filho!\nTCC Apresentado!');
} else {
    console.log('Parece o trabalho do meu filho!\nFail! Entao eh nataaaaal!');
}